fluss-common/src/main/java/com/alibaba/fluss/metadata/TableDescriptor.java

```diff
@@ -281,7 +283,27 @@ public boolean isDataLakeEnabled() {
     }
 
     public @Nullable MergeEngine getMergeEngine() {
-        return configuration().get(ConfigOptions.TABLE_MERGE_ENGINE);
+        MergeEngine mergeEngine = MergeEngine.create(properties);
+        if (mergeEngine != null) {
+            if (mergeEngine.getType() == MergeEngine.Type.VERSION) {
+                String column = mergeEngine.getColumn();
```

Comment:
Two comments about validate in this method
1. Actually, we only need to validate during creating the table. If validate in this method, every time we get a `TableDescriptor`  from server can try to get the merge engine of it, we have to do the validation.. Althoght it's not heavy, but unnesscassy..

2. If we may well support different data types for version in the future, I'd suggest to validate it in server side so that if support other data types, user won't need to bump it's client version.